spec/rubocop/cop/style/hash_transform_methods_spec.rb

```diff
@@ -0,0 +1,181 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Style::HashTransformMethods do
+  subject(:cop) { described_class.new }
+
```

Comment:
RuboCop supports Ruby 2.4 or lower. Can you add a tests that does not register an offense in `:ruby24`?